arch/arm/soc/atmel_sam0/samd/soc.h

```diff
@@ -0,0 +1,41 @@
+#ifndef _ATMEL_SAMD_SOC_H_
+#define _ATMEL_SAMD_SOC_H_
+
+#ifndef _ASMLANGUAGE
+
+#define DONT_USE_CMSIS_INIT
+#define DONT_USE_PREDEFINED_CORE_HANDLERS
+#define DONT_USE_PREDEFINED_PERIPHERALS_HANDLERS
+
+#include <zephyr/types.h>
+
+#if defined CONFIG_SOC_PART_NUMBER_SAMD21G18A
+#include <samd21g18a.h>
+#else
```

Comment:
We should add here all other part numbers which were imported to ext/hal/atmel/asf/sam0/include/samd/

Reply:
Done.